### DISCUSSION TITLE
Introduction of shape-it

### DISCUSSION THREAD

**Discussion by iwatobipen** [rdkit/shape-it#5] [General]
Hi,

Now new version of shape-it works not only OpenBabel but also RDKit and it can call from python as a library.
Thanks for Greg's and Hans's great work!
I think it will be useful information for many chemoinformatitian about this implementation.
So I would like to introduce this activity with some examples. Greg, do you have a plan to make a post about that in your RDKit blog?
Thanks,

Taka

**Comment by greglandrum** [comment]
I definitely agree that we should blog about it to let people know. I will definitely do a short post, but I want to have conda builds done for at least Mac and Linux first. I’m almost there with those.

i totally think that it makes sense for you to blog about it too, so please do so whenever you want. :-)

**Reply by iwatobipen** [reply]
It sounds great that you are developing conda package of shape-it.
Sure I would like to write a post after writing some examples which use shape-it.
Thanks.

**Reply by greglandrum** [reply]
The conda packages for the Mac and Linux are in the rdkit channel:
`conda install -c rdkit pyshapeit`
I'll do a PR this morning which includes the conda recipe.

**Comment by hansdewinter** [comment]
Hi all,

maybe it would be a good idea to include the original publication on the github page (Pharao was the original name of the code):

Taminau, J.; Thijs, G.; De Winter, H. (2008) ‘Pharao: Pharmacophore alignment and optimization’, J. Mol. Graph. Model. 27, 161-169

Regards,
Hans

**Reply by iwatobipen** [reply]
Thanks for your suggestion. It's a nice idea!
